src/stories/Library/opening-hours-editor/opening-hours-editor.scss

```
@@ -103,6 +103,25 @@
   height: 2rem;
 }
 
+.opening-hours-editor__loading {
+  position: absolute;
+  inset: 0;
+  background-color: rgba(255, 255, 255, 0.75);
+  display: flex;
+  gap: 10px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  z-index: 10;
+
+  > img {
```

I don't think > is necessary here, but not sure.

```suggestion
  img {
```